### ISSUE TITLE
Tensorboard displays graph but not loss graphs for Mask_RCNN 

### ISSUE THREAD

**Issue by soheilsadeghi90** [matterport/Mask_RCNN#1543]
I'm trying to train MaskRCNN using custom data and want to monitor its performance the same way as it's possible for tensorflow-object-detection api using a command like "tensorboard --logdir=<log_directory>". Using the current logs from training I can see the graph of the network but no scalar or loss values. Is there any way that I can get those results in a nice way?

**Comment by soheilsadeghi90** [comment]
The difference here is that the loss values are recorded per epoch; so before a full epoch is done, no results will be populated in tensorboard. It might be a way that one can request training loss to be reported within an epoch. If someone has the answer to this question, please let me know.
